<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Editor" name="editor">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>371</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>100</horstretch>
      <verstretch>100</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777214</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="outputTextEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>430</y>
      <width>441</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionClose"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
    <addaction name="actionInsert"/>
   </widget>
   <widget class="QMenu" name="menuCompile">
    <property name="title">
     <string>编译(&amp;C)</string>
    </property>
    <addaction name="actionCompile"/>
    <addaction name="actionRun"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuCompile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionUndo"/>
   <addaction name="actionCompile"/>
   <addaction name="actionRun"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭(&amp;C)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>替换(&amp;R)</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="text">
    <string>插入(&amp;I)</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="text">
    <string>编译(&amp;C)</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>运行(&amp;R)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Editor</class>
   <extends>QPlainTextEdit</extends>
   <header>editor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
